vendor/vendor.json

```diff
@@ -1,6 +1,6 @@
 {
 	"comment": "",
-	"ignore": "appengine test github.com/hashicorp/nomad/ github.com/hashicorp/terraform/backend",
+	"ignore": "test",
```

Comment:
why this change?

Reply:
No idea - govendor did it for me.  I'll put it back.